routers/api/v1/repo/issue_reaction.go

```diff
@@ -55,9 +57,20 @@ func GetIssueCommentReactions(ctx *context.APIContext) {
 		return
 	}
 
-	if !ctx.Repo.CanRead(models.UnitTypeIssues) && !ctx.User.IsAdmin {
-		ctx.Error(http.StatusForbidden, "GetIssueCommentReactions", errors.New("no permission to get reactions"))
-		return
+	if ctx.Repo.Repository.IsPrivate {
```

Comment:
Shouldn't we also check if organization is private?

Reply:
@lafriks can a repo be public when a org is private ?!?

Reply:
But I don't think it will be visible then, but that needs to be verified

Reply:
On UI I have no acces (404) on souch a repo (loged in/loged out)

Reply:
and the rest access permission is checed below (L66 R)

Reply:
Yes but if it will be public and org private it will not get to line 66